torch/fx/passes/graph_drawer.py

```diff
@@ -43,6 +44,7 @@
 }
 
 
+@compatibility(is_backward_compatible=False)
```

Comment:
cc @khabinov FYI I'm conservatively setting backward-compatibility for the APIs in the file to `False`. Please feel free to flip this to `True` if/when you feel comfortable providing stability guarantees for the API.